pkg/syncer/cnsoperator/controller/cnsfileaccessconfig/cnsfileaccessconfig_controller.go

```diff
@@ -228,6 +228,20 @@ func (r *ReconcileCnsFileAccessConfig) Reconcile(request reconcile.Request) (rec
 		// If not,
 		//   Do nothing (as there are other virtual machines on same external IP using this volume)
 		// Call RemoveClientVmFromIPList() to remove req.NodeID to list of nodes exposed by IP referencing this file volume.
+		volumeID, err := cnsoperatorutil.GetVolumeID(ctx, r.client, instance.Spec.PvcName, instance.Namespace)
```

Comment:
Can you update the TODO before this line, as you have refactored code quite a bit after that?

Reply:
We can remove the TODO, yes